scene/animation/animation_blend_tree.cpp

```diff
@@ -229,15 +229,17 @@ AnimationNodeSync::AnimationNodeSync() {
 ////////////////////////////////////////////////////////
 
 void AnimationNodeOneShot::get_parameter_list(List<PropertyInfo> *r_list) const {
-	r_list->push_back(PropertyInfo(Variant::BOOL, active));
-	r_list->push_back(PropertyInfo(Variant::BOOL, prev_active, PROPERTY_HINT_NONE, "", PROPERTY_USAGE_NONE));
+	r_list->push_back(PropertyInfo(Variant::BOOL, active, PROPERTY_HINT_NONE, "", PROPERTY_USAGE_EDITOR | PROPERTY_USAGE_STORAGE | PROPERTY_USAGE_READ_ONLY));
+	r_list->push_back(PropertyInfo(Variant::INT, request, PROPERTY_HINT_ENUM, ",Fire,Abort"));
```

Comment:
The first one has no name?

Reply:
Yes. If there is a string here, the BlendTreeEditor pull-down will display something like "None". I think it's fine to leave it blank, or though it could be displayed as "--".

Reply:
Alright, I'm not familiar enough with the API to say what would make sense but UX wise it doesn't seem immediately obvious why there's an empty drop down list, and what the empty option does compared to the other two.

But I trust your judgment on this. Can always be improved later on if users do report this as confusing :)